doc/api/cli.md

```diff
@@ -751,6 +751,10 @@ Enables report to be generated on uncaught exceptions. Useful when inspecting
 the JavaScript stack in conjunction with native stack and other runtime
 environment data.
 
+### `--set-output-dir=directory`
+
+Set the directory for which all output files should be written to.
```

Comment:
```suggestion
Set the directory for which all diagnostic output files should be written to.
```

Also, would be good to indicate that this is cwd by default.

Reply:
Double linking can be painful to maintain, but after the beginning, there shouldn't be too much churn (in the sense of features that are affected by this option), so I suggest this should link to the options that it affects, and the features it affects should link back to here.

Reply:
@sam-github what do you mean by double linking? is it just as simple as listing the affected options or do you mean put an actual link to the other options?

Reply:
Something like, here:
```
Affects the default value of:
* [--report-dir](link to the --report-dir doc anchor)
* [--cpu-prof-dir](...)
```

Then in the docs for `--report-dir` (and all the others) say `defaults to CWD, but that can be changed using the [--diagnostic-dir](link to the doc anchor) command line option`.

That's the double link, the --feature-dir docs link to the global option, the global option links to all the --feature-dir options it sets the default of.